<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-78D4HDH" ignored="0" name="Test" tests="11" failures="5" timestamp="11 Oct 2018 12:28:25 GMT" time="0.048" errors="0">
  <testcase name="method1" time="0.009" classname="basicsTestNG.TestNGAlwaysRun">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at basicsTestNG.TestNGAlwaysRun.method1(TestNGAlwaysRun.java:13)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- method1 -->
  <testcase name="method2" time="0.0" classname="basicsTestNG.TestNGAlwaysRun"/>
  <testcase name="method3" time="0.001" classname="basicsTestNG.TestNGAlwaysRun"/>
  <testcase name="methodA" time="0.001" classname="assertionTestNG.AssertionExample"/>
  <testcase name="methodB" time="0.0" classname="assertionTestNG.AssertionExample">
    <failure type="java.lang.AssertionError" message="Mismatched expected [11] but found [10]">
      <![CDATA[java.lang.AssertionError: Mismatched expected [11] but found [10]
at assertionTestNG.AssertionExample.methodB(AssertionExample.java:19)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- methodB -->
  <testcase name="methodC" time="0.0" classname="assertionTestNG.AssertionExample"/>
  <testcase name="methodD" time="0.001" classname="assertionTestNG.AssertionExample">
    <failure type="java.lang.AssertionError" message="Not Matching expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Not Matching expected [true] but found [false]
at assertionTestNG.AssertionExample.methodD(AssertionExample.java:32)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- methodD -->
  <testcase name="methodE" time="0.0" classname="assertionTestNG.AssertionExample"/>
  <testcase name="methodF" time="0.0" classname="assertionTestNG.AssertionExample"/>
  <testcase name="methodG" time="0.001" classname="assertionTestNG.AssertionExample">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at assertionTestNG.AssertionExample.methodG(AssertionExample.java:52)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- methodG -->
  <testcase name="methodH" time="0.0" classname="assertionTestNG.AssertionExample">
    <failure type="java.lang.AssertionError" message="Fail TestCase">
      <![CDATA[java.lang.AssertionError: Fail TestCase
at assertionTestNG.AssertionExample.methodH(AssertionExample.java:59)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- methodH -->
</testsuite> <!-- Test -->
